posthog/models/remote_config.py

```diff
@@ -177,8 +179,12 @@ def build_config(self):
                 config["quotaLimited"] = ["recordings"]
                 config["sessionRecording"] = False
 
-        config["surveys"] = True if team.surveys_opt_in else False
         config["heatmaps"] = True if team.heatmaps_opt_in else False
+        surveys_response = get_surveys_response(team)
+        config["surveys"] = surveys_response["surveys"]
+        if surveys_response["survey_config"]:
+            config["survey_config"] = surveys_response["survey_config"]
```

Comment:
```suggestion
        survey_config = surveys_response["survey_config"]
        if survey_config:
            config["survey_config"] = survey_config
```

nitpick but I tend to assign to a var so we don't look up the map twice, it's more important when the map has many keys so not a must here, your call.

Reply:
I'd debate this has almost zero perf impact 🤷 will ignore in favour of getting this in

Reply:
agree!

> it's more important when the map has many keys so not a must here

:)